main.nf

```diff
@@ -1668,6 +1668,7 @@ if (!params.skipAlignment) {
             --isoform_level=$params.isoform_level \
             --mcmc_iter=$params.mcmc_iter \
             --mcmc_warmup=$params.mcmc_warmup \
+            --zeroes_threshold=$params.zeroes_threshold \
```

Comment:
Adds newly defined parameter of `rsem2hbadeals.R` in the Nextflow process 'hbadeals'.